docs/main/ios/spm.md

```diff
@@ -59,15 +59,79 @@ Then let's sync the web app. This will add the App plugin SPM to the iOS project
 
 You can now use the App plugin normally.
 
-<em>More details coming soon</em>
+## Using SPM in an existing Capacitor Project
+
+First, ensure you have a backup of the current state of your project, either in source control or elsewhere.
+
+### Deleting your iOS Directory
+
+If you **have not manually changed your Xcode Project at all**, one option to migrate is to delete the `ios` directory and then run `npx cap add ios --packagemanager SPM`. This will remove the CocoaPods template project and replace it with the SPM template project.  
+### Using our migration tool
+
+The Capacitor CLI has a command to help migrate from CocoaPods to Swift Package Manager. However, one manual step is still required. In addition, projects with Cordova plugins will not be migrated correctly and neither will projects that use plugins that do not have SPM versions available.
+
+To start, run `npx cap spm-migration-assistant` in the root of your project.
+
+This tool will:
+  - Run `pod deintegrate` removing CocoaPods
+  - Delete the `Podfile`, `App.xcworkspace`, and `Podfile.lock`
+  - Create a `CapApp-SPM` directory with the needed files
+  - Generate a `Package.swift` from your plugins, and warn you if any can't included. 
+
+After this is run, run a `npx cap sync` again.
```

Comment:
As mentioned in the CLI PR, this restores the incompatible plugins, the command should run `update` programmatically and `update` should handle the plugins and should not be ran manually by users here.

Reply:
I fixed the PR so update works.